README.rst

```diff
@@ -551,7 +551,7 @@ whose names contain the substring :code:`substr`:
 .. code::
 
     {
-        Animal {
+        Animal @fold {
```

Comment:
The `@fold` should actually be one level deeper, on `out_Animal_ParentOf`. Putting it here should fail to compile since it violates:

```
-  All @output fields within a @fold traversal must be present
   at the innermost scope. It is invalid to expand vertex fields within
   a @fold after encountering an @output directive.
```